src/site/content/en/fast/content-delivery-networks/index.md

```diff
@@ -0,0 +1,286 @@
+
+
+#### Brotli from origin
+
+Brotli from origin is when a CDN serves resources that were Brotli-compressed by the origin. Although this may seem like a feature that all CDNs should be able to support out of the box, it requires that a CDN be able to cache multiple versions (i.e. gzip-compressed and Brotli-compressed) of the resource corresponding to a given URL.
+
+
+#### Automatic Brotli compression
+
+Automatic Brotli compression is when resources are Brotli compressed by the CDN. This is usually implemented by using offline processing. The first time that a resource is requested it is served using "good enough" compression - for example, Brotli-5. Meanwhile, the resource will be compressed offline using a more powerful but far slower compression level - for example, Brotli-11. Once this compression completes, the more compressed version will be cached and used for subsequent requests.
+
+
+#### Compression best practices
+
+Sites that want to maximize performance should apply Brotli compression at both their origin server and CDN. Brotli compression at the origin minimizes the transfer size of resources that can't be served from the cache. To prevent delays in serving requests, the origin should compress dynamic resources using a fairly conservative compression level - for example, Brotli-4; static resources can be compressed using Brotli-11.
+
+
+### TLS 1.3
+
+TLS 1.3 is the newest version of [Transport Layer Security (TLS)](https://en.wikipedia.org/wiki/Transport_Layer_Security), the cryptographic protocol used by [HTTPS](https://en.wikipedia.org/wiki/HTTPS). TLS 1.3 is more secure than TLS 1.2 and also shortens the TLS handshake from two roundtrips to one. For connections using HTTP/1 or HTTP/2, shortening the TLS handshake to one roundtrip effectively reduces connection setup time by 33%.
+
+<figure class="w-figure">
+  <img src="./cdn3.png" alt="Comparison of the TLS 1.2 and TLS 1.3 handshakes" class="w-screenshot">
+</figure>
+
+### HTTP/2 & HTTP/3
+
+HTTP/2 and HTTP/3 both provide performance benefits over HTTP/1. Of the two, HTTP/3 offers greater _potential_ performance benefits. HTTP/3 isn't fully standardized yet, but it will be widely [supported](https://caniuse.com/#feat=http3) once this occurs.
+
+
+#### HTTP/2
+
+If your CDN hasn't already enabled [HTTP/2](https://en.wikipedia.org/wiki/HTTP/2) by default, you should consider turning it on. HTTP/2 provides multiple [performance benefits](https://hpbn.co/http2) over HTTP/1 and is [supported](https://caniuse.com/#feat=http2) by all major browsers. Performance features of HTTP/2 include: [multiplexing](https://hpbn.co/http2/#request-and-response-multiplexing), [stream prioritization](https://hpbn.co/http2/#stream-prioritization), [server push](https://developers.google.com/web/fundamentals/performance/http2), and [header compression](https://blog.cloudflare.com/hpack-the-silent-killer-feature-of-http-2/).
```

Comment:
Maybe use https://almanac.httparchive.org/en/2019/http2 as more up to date review of http/2 and its usage.